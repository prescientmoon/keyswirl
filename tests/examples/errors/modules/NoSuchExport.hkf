module exporting (aaa, aa, bb, cc) -- The compiler will complain cc is not in scope

import errors.modules.helper (aa, bb, cc) as CC -- the compiler will tell you cc isn't a valid export
import errors.modules.helper 

alias aaa = [CC.aa, bb]
