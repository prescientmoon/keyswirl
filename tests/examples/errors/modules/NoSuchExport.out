[error VarNotInScope]: Undefined variable
     +--> errors.modules.NoSuchExport@1:32-1:34
     |
   1 | module exporting (aaa, aa, bb, cc) -- The compiler will complain cc is not in scope
     :                                ^- 
     :                                `- Undefined variable cc
-----+

[error NoSuchExport]: No such export
     +--> errors.modules.NoSuchExport@3:39-3:41
     |
   3 | import errors.modules.helper (aa, bb, cc) as CC -- the compiler will tell you cc isn't a valid export
     :        ^--------------------          ^- 
     :        |                              `- No declaration with this name found in the given module.
     :        `- This is the module you tried importing "cc" from
-----+
